src/bitcoind.cpp

```diff
@@ -19,6 +19,9 @@
 #include <util/strencodings.h>
 #include <util/system.h>
 #include <util/threadnames.h>
+#include <util/translation.h>
```

Comment:
translations are only for the gui, this should never be added here

Reply:
> translations are only for the gui, this should never be added here

I haven't looked at the context for this comment yet, but in general translations are desirable when writing error messages to stderr, right? My understanding is we only want to avoid translating very technical messages, rare messages, and messages sent to debug.log.

Reply:
> translations are only for the gui, this should never be added here

That is correct. But without `#include <util/translation.h>` the linker fires an error:
```
  CXXLD    bitcoind
libbitcoin_server.a(libbitcoin_server_a-init.o): In function `_(char const*)':
/home/hebasto/github/bitcoin/src/./util/translation.h:39: undefined reference to `G_TRANSLATION_FUN[abi:cxx11]'
```